.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 language: ruby
 
 rvm:
-  - 2.5.3
```

Comment:
@Judahmeek I don't think the .travis file is used any more.

Reply:
Neither Travis or CircleCI are used anymore. Would you like us to remove those files?

Reply:
Circle CI might be a good example to keep. I'd say that we could move the files to an /archive directory with a README? Or maybe one can find in the git history if we document that this in a section on CI for ROR?